lib/Runtime/Types/ES5ArrayTypeHandler.cpp

```diff
@@ -371,7 +371,7 @@ namespace Js
         // Reject if we need to grow non-writable length
         if (!CanSetItemAt(arr, index))
         {
```

Comment:
@jianchun I checked some others cases like Object.defineProperty which doesn't enter the other occurrences in the code. How about check in this with a comment? Probably extend it later.